### PR TITLE
Fixes #1906 - refactor template about.html

### LINKED ISSUE #1906: refactor template: about.html
Refactor following:
https://github.com/webcompat/webcompat.com/blob/master/webcompat/templates/about.html

with:
.grid > .grid-row > .grid-cell.x3

Please be aware to add the placeholders for links as they are in the old templates. 
Especially for the JS.

### PR DESCRIPTION
**Pull request by zoepage**
- [x] add changed on copy
- [x] add bios of team

### CODE REVIEW

File: webcompat/templates/about.html [hunk 1]
```diff
@@ -70,13 +71,244 @@ <h4 class="headline-3">Reaching Out</h4>
...
+      </article>
+
+<!-- BEA Rizental -->
+      <article class="grid-row bio">
+        <div class="grid-cell x1-fix">
+          <img src="https://avatars3.githubusercontent.com/u/25176023" alt="Beatriz Rizental" class="bio-img">
+        </div>
+        <div class="grid-nested x1">
+          <div class="grid-cell bio-desc x1">
+            <h4 class="headline-2 highlight">Beatriz Rizental</h4>
```
Comment (miketaylr) [inline webcompat/templates/about.html]: @brizental would you prefer Bea here, rather than Beatriz?

File: webcompat/templates/about.html [hunk 2]
```diff
@@ -70,13 +71,244 @@ <h4 class="headline-3">Reaching Out</h4>
...
+          <div class="grid-cell bio-desc x1">
+            <h4 class="headline-2 highlight">Adam Stevenson</h4>
+            <h4 class="headline-3">Canada</h4>
+            <p>
+              Works on webcompat bugs and does outreach to sites / developers. Enjoys music, biking,
+              travel and sometimes photos.
+            </p>
+          </div>
+          <p class="grid-cell bio-links x1">
+            <a class="bio-icon-twitter" href="http://twitter.com/adamopenweb" title="adamopenweb on twitter">twitter@adamopenweb </a> <br>
```
Comment (miketaylr) [inline webcompat/templates/about.html]: is this pattern on purpose?: `twitter@adamopenweb` should it just be `@adamopenweb`?

File: webcompat/templates/about.html (hunk 1 above)
Comment (brizental) [reply webcompat/templates/about.html]: Ahh, yes!

Comment (miketaylr) [reply webcompat/templates/about.html]: @zoepage can you change it to Bea, please? Thanks!

File: webcompat/templates/about.html (hunk 2 above)
Comment (zoepage) [reply webcompat/templates/about.html]: Yes, it is. Can also add icons if this is something you'd prefer.

Comment (miketaylr) [reply webcompat/templates/about.html]: It's just a little confusing, but I haven't looked at what the design is. Is the text visible?

Comment (zoepage) [reply webcompat/templates/about.html]: screen shot 2017-12-05 at 10 17 21 am

File: webcompat/templates/about.html (hunk 1 above)
Comment (zoepage) [reply webcompat/templates/about.html]: sure.

File: webcompat/templates/about.html (hunk 2 above)
Comment (miketaylr) [reply webcompat/templates/about.html]: Cool, OK. As long as it's on purpose I trust your judgement here (especially since they are hyperlinks). ^__^

**Pull request merged**